merging-configmaps-gitops/README.md

```diff
@@ -0,0 +1,83 @@
+# Merging config in a gitops context
+
+Our app platform utilizes configmaps / secrets heavily to pass configuration into apps.
+
+*The following RFC will focus on configmaps but the same problem and solution is also applicable for secrets!*
+
+These configmaps are often either shared between multiple instances of the same app or have shared values between apps.
+In gitops the intended solution for both of these cases is to use `bases` and `overwrites` which is achieved through `kustomize`.
```

Comment:
Can you show what this looks like here or is it too much for an RFC?

Reply:
I decided to simply add a link.